addons/pos_self_order_stripe/controllers/orders.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from odoo import http, fields
+from odoo.http import request
+from odoo.addons.pos_self_order.controllers.orders import PosSelfOrderController
+
+
+class PosSelfOrderControllerStripe(PosSelfOrderController):
+    @http.route("/pos-self-order/stripe-connection-token/", auth="public", type="json", website=True)
+    def get_stripe_creditentials(self, access_token, payment_method_id):
+        # stripe_connection_token
+        pos_config, _ = self._verify_authorization(access_token, "", False)
+        payment_method = pos_config.env['pos.payment.method'].browse([payment_method_id])
+        return payment_method.stripe_connection_token()
+
+    @http.route("/pos-self-order/stripe-capture-payment/", auth="public", type="json", website=True)
+    def stripe_capture_payment(self, access_token, order_access_token, payment_intent_id, payment_method_id):
+        pos_config, _ = self._verify_authorization(access_token, "", False)
+        stripe_confirmation = pos_config.env['pos.payment.method'].stripe_capture_payment(payment_intent_id)
+        order = pos_config.env['pos.order'].search([('access_token', '=', order_access_token)])
```

Comment:
shouldn't we check that the config matches the order?

Reply:
In the event that a terminal or config wants to validate a payment, I prefer to be sure that the request goes through and marks the order as paid, because at that point, the money has already been taken from the customer's account. So I check that the payment method is on the pos_config, but I don't think it's necessary to check that the order is linked to it.

Reply:
Could you add a docstring for those controllers to briefly explain at which step of the process they are used?
Because here, it means that somebody could reuse the config and payment method of a past order to pay a new order made on another kiosk. Is that a plausible/bad scenario?

Reply:
You are right, isn't a possible scenario, I will add the verification.